please.sh

```diff
@@ -761,6 +761,17 @@ build_and_test_64 () {
 	test $# = 0 ||
 	die "Expected no argument, got $#: %s\n" "$*"
 
+	clean_t='perl -pe "s/ok \d+ # skip/skipped:/" |
```

Comment:
I would like to add this with a flag to turn it off. To that end, I would make the `| ` part of the `clean_t`. And I would rename `clean_t` to `filter_make_test`.